src/Backups/BackupImpl.h

```diff
@@ -81,8 +81,9 @@ class BackupImpl : public IBackup
     size_t copyFileToDisk(const String & file_name, DiskPtr destination_disk, const String & destination_path, WriteMode write_mode) const override;
     size_t copyFileToDisk(const SizeAndChecksum & size_and_checksum, DiskPtr destination_disk, const String & destination_path, WriteMode write_mode) const override;
     void writeFile(const BackupFileInfo & info, BackupEntryPtr entry) override;
-    void finalizeWriting() override;
```

Comment:
Looks like the declaration of `removeAllFilesAfterFailure` is not removed

Reply:
I've removed it now